extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -32,6 +32,15 @@ import {setStyle} from '../../../src/style';
 import {loadPromise} from '../../../src/event-helper';
 import {getHtml} from '../../../src/get-html';
 import {removeElement} from '../../../src/dom';
+import {getServiceForDoc} from '../../../src/service';
+import {MessageType} from '../../../src/3p-frame-messaging';
+import {isExperimentOn} from '../../../src/experiments';
+import {
+  installPositionObserverServiceForDoc,
```

Comment:
Since it's still under experiment. I would suggest we add a presubmit check for `installPositionObserverServiceForDoc`?